jkube-kit/enricher/generic/src/test/java/org/eclipse/jkube/enricher/generic/DefaultNamespaceEnricherTest.java

```diff
@@ -146,62 +129,38 @@ public void enrichWithPropertiesInKubernetesShouldAddNamespaceWithStatus() {
         .hasFieldOrPropertyWithValue("metadata.namespace", "example");
   }
 
-  @Test
-  public void enrichWithPropertiesInKubernetesShouldAddProjectWithStatus() {
+  @DisplayName("enrich with")
+  @ParameterizedTest(name = "{0}")
+  @MethodSource("enrichWithTestData")
+  void enrich(String description, HasMetadata item, PlatformMode platformMode, String expectedNamespace, String expectedName) {
```

Comment:
one more time :sweat_smile: 
As a rule of thumb, if you need to add a `description` field to a parameterized test, then very likely it's not a good idea to make it parameterized.